### PR TITLE
Refactor functions that print commands (Part 2)

### PR DESCRIPTION
**Pull request by abdoo8080**
This PR is a step towards migrating commands to the Term/Sort level. It replaces the dynamic casts for printing commands with direct calls to corresponding functions. Those functions now take node level arguments instead of commands to make them available for internal code.

### CODE REVIEW

File: src/printer/tptp/tptp_printer.cpp [hunk 1]
```diff
@@ -91,6 +82,210 @@ void TptpPrinter::toStream(std::ostream& out, const UnsatCore& core) const
...
+                                       const std::string& flag,
+                                       SExpr sexpr) const
+{
+  printUnknownCommand(out, "set-option");
+}
+
+void TptpPrinter::toStreamCmdGetOption(std::ostream& out,
+                                       const std::string& flag) const
+{
+  printUnknownCommand(out, "get-option");
```
Comment (ajreynol) [inline src/printer/tptp/tptp_printer.cpp:246]: Can you omit any overrides that just print unknown?

Comment (abdoo8080) [reply src/printer/tptp/tptp_printer.cpp]: Done!

File: src/printer/cvc/cvc_printer.cpp [hunk 2]
```diff
@@ -139,7 +139,7 @@ void CvcPrinter::toStream(
   if(n.getMetaKind() == kind::metakind::CONSTANT) {
     switch(n.getKind()) {
     case kind::BITVECTOR_TYPE:
-      out << "BITVECTOR(" << n.getConst<BitVectorSize>().d_size << ")";
+      out << "BITVECTOR(" << n.getConst<BitVectorSize>().d_size << ')';
```
Comment (ajreynol) [inline src/printer/cvc/cvc_printer.cpp:142]: Why this change?

File: src/smt/command.cpp [hunk 3]
```diff
@@ -430,6 +492,23 @@ Command* CheckSatCommand::clone() const
 
 std::string CheckSatCommand::getCommandName() const { return "check-sat"; }
 
+void CheckSatCommand::toStream(std::ostream& out,
+                               int toDepth,
+                               bool types,
+                               size_t dag,
+                               OutputLanguage language) const
+{
+  if (d_expr.isNull())
```
Comment (ajreynol) [inline src/smt/command.cpp:501]: I think I'd prefer to make `Printer::toStreamCmdCheckSat` robust to whether the expression was null and avoid the case split here.

File: src/printer/cvc/cvc_printer.cpp (hunk 2 above)
Comment (abdoo8080) [reply src/printer/cvc/cvc_printer.cpp]: I have a personal preference for characters. Though, it looks like the change affected code unrelated to my PR. Reverted.

File: src/smt/command.cpp [hunk 4]
```diff
@@ -105,6 +105,32 @@ std::ostream& operator<<(std::ostream& out, BenchmarkStatus status)
   }
 }
 
+std::vector<Node> exprVectorToNodes(const std::vector<Expr>& exprs)
```
Comment (ajreynol) [inline src/smt/command.cpp:108]: Can you mark this with `// !!!!! temporary until commands are migrated` ?
